SU2_CFD/src/numerics/ReadBFMInput.cpp

```diff
@@ -0,0 +1,323 @@
+/*!
+ * \file ReadBFMInput.cpp
+ * \brief 
+ * \author E.C. Bunschoten
+ * \version 7.1.0 "Blackbird"
+ *
+ * SU2 Project Website: https://su2code.github.io
+ *
+ * The SU2 Project is maintained by the SU2 Foundation
+ * (http://su2foundation.org)
+ *
+ * Copyright 2012-2020, SU2 Contributors (cf. AUTHORS.md)
+ *
+ * SU2 is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * SU2 is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with SU2. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "../../include/numerics/ReadBFMInput.hpp"
+
+ReadBFMInput::ReadBFMInput(CConfig *config, string file_inputname) 
```

Comment:
```suggestion
#include "../../include/numerics/ReadBFMInput.hpp"

CReadBFMInput::CReadBFMInput(CConfig *config, string file_inputname) 
```
For a reader this is in an odd place, there is a marker profile reader floating around, you could put them together in a "filereader" folder.